docs/archetypes/generic-subscription.md

```diff
@@ -314,6 +322,49 @@ This example configures:
 }
 ```
 
+## Recommended Parameter Property Updates
+
+### Service Health Alerts
+
+Update the "serviceHealthAlerts": {"receivers": {"app": [], "email": []} } properties with specific email addresses and phone numbers as required.
+
+![Generic Subscription: Service Health Alerts](../../docs/media/archetypes/service-health-alerts-receivers.jpg) 
+
+### Security Center
+
+Change the "securityCenter": {"value": {"email": "", "phone": ""} properties with specific email and address values to reflect your actual point of contact.
+
+![Generic Subscription: Security Center](../../docs/media/archetypes/security-center-contact-info.jpg)
+
+### Subscription Role Assignments
+
+Modify the two "subscriptionRoleAssignments: {"value": ["securityGroupObjectIds": []]}" properties with your specific unique object ids of the respective groups for the **Contributor** built-in
+and **Custom Role: Landing Zone Application Owner** roles for this landing zone subscription.
+
+![Generic Subscription: Subscription Role Assignments](../../docs/media/archetypes/subscription-role-assignments.jpg)
+
+### Resource Tags and Preferred Naming Convention
+
+1. Specify the desired custom values for the following tags: "resourceTags": {"value": {"ClientOrganization": "","CostCenter": "", "DataSensitivity": "", "ProjectContact": "", "ProjectName": "", "TechnicalContact": ""}".
+You may also include any additional name value pairs of tags required.
+
+2. Addtionally, you can also customize default resource name values with any specific preferred naming conventions, as indicated by the item **2** circles shown below.
+
+![Generic Subscription: Tags and Naming Conventions](../../docs/media/archetypes/resource-tags-and-naming-conventions.jpg)
+
+### Hub Virtual Network ID
+
+**IMPORTANT**
+
+To avoid a failure when running any of the connectivity pipelines, the subscriptionId segment value of the "hubNetwork": {"value": } string (item **1**), must be updated from it's default value to the specific hubNetwork subscriptionId that was 
+actually deployed previously, so this subscription as a spoke can be peered to it.
```

Comment:
can you change `so this subscription as a spoke can be peered to it` to

so that the virtual network in this spoke subscription can be VNET Peered to the Hub Network.